thunder/core/prims.py

```diff
@@ -4079,39 +4077,6 @@ def maybe_expand_seq(seq, ndim):
 )
 
 
-def embedding_meta(
-    a: TensorProxy, /, weight, *, padding_idx=-1, max_norm=None, norm_type=2.0, scale_grad_by_freq=False, sparse=False
-) -> TensorProxy:
-    # TODO: canonicalize and validating padding idx with weight.shape[0]
-
-    if max_norm is not None:
-        raise NotImplementedError
```

Comment:
The `max_norm` argument was not supported. I don't know the reason for this. I think it should work and I'll update OpInfo samples to test it.

Reply:
`max_norm` argument when active modifies the `weight` argument inplace. That's probably the reason this argument raised NotImplementedError. I'll keep the behavior as is in this PR.
Created an issue for tracking completion https://github.com/Lightning-AI/lightning-thunder/issues/1699.